repos/spack_repo/builtin/packages/py_protobuf/package.py

```diff
@@ -49,6 +49,7 @@ class PyProtobuf(PythonPackage):
     version("3.15.5", sha256="be8a929c6178bb6cbe9e2c858be62fa08966a39ae758a8493a88f0ed1efb6097")
     version("3.15.1", sha256="824dbae3390fcc3ea1bf96748e6da951a601802894cf7e1465e72b4732538cab")
     version("3.13.0", sha256="6a82e0c8bb2bf58f606040cc5814e07715b2094caeba281e2e7d0b0e2e397db5")
+    version("3.12.4", sha256="c99e5aea75b6f2b29c8d8da5bdc5f5ed8d9a5b4f15115c8316a3f0a850f94656")
```

Comment:
You don't need to add this version in order to have it as a minimum in for the package in `py-geopmdy`. The goal of specifying the minimum was to ensure no attempt is made to use the older versions.

I recommend dropping this version.